### PR TITLE
[iOS] Allow to send email with logs on iOS

### PR DESCRIPTION
**Pull request by rasom**
fetching status-go logs using `StatusgoExportNodeLogs()` will be added in the separate PR, requires status-go update so might take some extra time

status: ready

### CODE REVIEW

File: modules/react-native-status/ios/RCTStatus/RCTStatus.m [hunk 1]
```diff
@@ -227,11 +228,56 @@ - (void)handleSignal:(NSString *)signal
...
+    [fileManager removeItemAtPath:zipFile.path error:nil];
+    
+    NSURL *logsFolderName = [rootUrl URLByAppendingPathComponent:@"logs"];
+    
+    if (![fileManager fileExistsAtPath:logsFolderName.path])
+        [fileManager createDirectoryAtPath:logsFolderName.path withIntermediateDirectories:YES attributes:nil error:&error];
+    
+    NSURL *dbFile = [logsFolderName URLByAppendingPathComponent:@"db.json"];
+#if DEBUG
+    NSString *networkDirPath = @"ethereum/mainnet_rpc_dev";
```
Comment (mandrigin) [inline modules/react-native-status/ios/RCTStatus/RCTStatus.m:253]: hmm, why do we limit to mainnet only?

File: modules/react-native-status/ios/RCTStatus/RCTStatus.m [hunk 2]
```diff
@@ -227,11 +228,56 @@ - (void)handleSignal:(NSString *)signal
...
+#else
+    NSString *networkDirPath = @"ethereum/mainnet_rpc";
+#endif
+    
+    NSURL *networkDir = [rootUrl URLByAppendingPathComponent:networkDirPath];
+    NSURL *originalGethLogsFile = [networkDir URLByAppendingPathComponent:@"geth.log"];
```
Comment (mandrigin) [inline modules/react-native-status/ios/RCTStatus/RCTStatus.m:259]: why don't we use `ExportNodeLogs` here? https://github.com/status-im/status-go/blob/develop/mobile/status.go#L640

**Review by mandrigin — DISMISSED** [review]
a few questions

File: modules/react-native-status/ios/RCTStatus/RCTStatus.m (hunk 1 above)
Comment (rasom) [reply modules/react-native-status/ios/RCTStatus/RCTStatus.m:253]: Yes, till we fix that encoding in status-go api method. That’s rather temporary thing to make sure we get at least some data in email on iOS

File: modules/react-native-status/ios/RCTStatus/RCTStatus.m (hunk 2 above)
Comment (rasom) [reply modules/react-native-status/ios/RCTStatus/RCTStatus.m:259]: Because of encoding, I wrote on Friday, forgot to remind about this on call today

**Review by flexsurfer — APPROVED** [review]

**Review by cammellos — APPROVED** [review]

**Comment by churik** [comment]
@rasom 
tested on https://i.diawi.com/TBj6Ab 
When extracting logs I see encrypted `geth.log` with error inside and `db.json`
Expected - `Status.log` and `db.json`
Logs: status_logs.zip](https://github.com/status-im/status-react/files/3173966/status_logs.zip)
OS: IOS 11.4.1 (IPhone 7)

**Comment by rasom** [comment]
@churik `Status.log` on iOS and Desktop is added in the separate pr https://github.com/status-im/status-react/pull/8081

**Comment by rasom** [comment]
> When extracting logs I see encrypted geth.log

@churik are you sure you installed build №10? I can't reproduce it.

**Comment by churik** [comment]
@rasom some glitch in https://github.com/status-im/status-react/pull/8124#issuecomment-489119153 (last builds were #9)
Tested on #10 build on IOS 11.4.1 (IPhone 7):
- that can send logs (mail app is used, address in `error-reports@status.im` )
- that can change log level, network settings and send logs;
- .zip contains geth.log and db.json

Checked that doesn't affect to Android, so I can send logs.

**Pull request merged**